products/prisma-cloud/api/cspm/api-integration-config.md

```diff
@@ -10,6 +10,78 @@ The request body for some of the endpoints includes an `integrationConfig` param
 
 Note that most external systems require some configuration before you can use the Prisma Cloud API endpoints to add an integration to that system. For more details, see [Prisma Cloud Integrations](https://docs.prismacloud.io/en/classic/cspm-admin-guide/configure-external-integrations-on-prisma-cloud/prisma-cloud-integrations). If you are upgraded to the Darwin release, see [Prisma Cloud Integrations](https://docs.prismacloud.io/en/enterprise-edition/content-collections/administration/configure-external-integrations-on-prisma-cloud/prisma-cloud-integrations).
 
+
+### AWS Security Hub
+
+Prisma Cloud integrates with AWS Security Hub for centralized visibility into security and compliance risks associated with your cloud assets on the AWS Security Hub console.
+
+As a part of the integration, Prisma Cloud monitors your AWS cloud assets. It sends alerts about resource misconfigurations, compliance violations, network security risks, and anomalous user activities directly to the AWS Security Hub console providing a centralized and comprehensive view of the cloud assets deployed on your AWS accounts.
+
+
+Note: Prisma Cloud integration with AWS Security Hub is not supported for `US Gov Cloud` regions.
+
+
+#### Add, Update, or Test an AWS Security Hub Integration
+
+
+To add an AWS Security Hub integration, make your request as described in
+[Add Integration](/prisma-cloud/api/cspm/create-integration-v-1). As part of the request body, the `integrationType` parameter is `aws_security_hub`, and the `integrationConfig` parameter contains the following key/value pairs.
+
+
+| Key            | Value Description                                                                                                 | Value Type | Default Value or Required |
+| -------------- | ----------------------------------------------------------------------------------------------------------------- | ---------- | ------------------------- |
+| region         | List of Regions where Prisma Cloud on AWS Security Hub is enabled for your cloud account | List of Objects<br/>{<br/>String name; (Required)<br/>String apiIdentifier; (Required)<br/>String cloudType;<br/>String sdkId;<br/>Boolean enabled;<br/>}    | Required               |
```

Comment:
Should the attributes be formatted as code? e.g. `region`

Reply:
yes. So that developers can use the same attribute and relate to it.